packages/wonder-blocks-badge/src/components/__tests__/status-badge.test.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import {render, screen} from "@testing-library/react";
+import {StatusBadge} from "../status-badge";
+
+describe("StatusBadge", () => {
```

Comment:
I included specific unit tests for the main props of StatusBadge (`label` and `icon`). 

Another reason for doing this is instead of including it in the shared tests is because the API related to icons will be different for the other badge types. `GemBadge` and `StreakBadge` have the icon predefined so the icon prop will be `showIcon?: boolean` instead of `icon?: ReactElement`